requirements/base.txt

```diff
@@ -20,10 +20,15 @@ Markdown==2.6.8
 numpy==1.12.0
 openpyxl==2.4.1
 pandas==0.19.2
+# patsy is for statsmodels
+patsy==0.4.1
 Pillow==3.2.0
 psycopg2==2.6.1
 pytz==2016.10
 readline==6.2.4.1
+# necessary for kendall / spearman correlations, default is pearson on pandas
+scipy==0.19.0
+statsmodels==0.8.0
```

Comment:
explain it's needed for OLS (but ... should you hold off since you're not using it yet)?